### DISCUSSION TITLE
2023-05-11 ミーティングアジェンダ

### DISCUSSION THREAD

**Discussion by azu** [asciidwango/js-primer#1663] [Meeting]
2023-05-11のミーティングアジェンダ置き場

**Comment by azu** [comment]
修正予定のIssue

- https://github.com/asciidwango/js-primer/issues/1656
- https://github.com/asciidwango/js-primer/issues/1657
- https://github.com/asciidwango/js-primer/issues/1659
- https://github.com/asciidwango/js-primer/issues/1660

**Comment by azu** [comment]
## その他

- marked@5 がリリースされた
  - https://github.com/markedjs/marked/releases/tag/v5.0.0
  - 外部依存はないので、まあ4のままでもいいかな
  - https://github.com/markedjs/marked/blob/v4.3.0/package.json
- https://github.com/asciidwango/js-primer/blob/ecd2e66c12d5aed5e843ca4326e36e3cb11fb0dd/source/use-case/nodecli/argument-parse/README.md?plain=1#LL34C5-L34C5


ここにユーザー名が入ってることに気づいた。(どちらでも良さそう)

**Reply by lacolaco** [reply]
たしかに。なんかダミーっぽいファイルパスに置き換えたい気持ち

**Reply by lacolaco** [reply]
`  '/Users/.../nodecli/main.js', // 実行したスクリプトファイルのパス` とかでいいかな　組版的にも幅ほとんど変わらない

**Reply by lacolaco** [reply]
ファイルパスは直さないことにした

**Reply by azu** [reply]
marked 本は4のままで

**Comment by azu** [comment]
出版日 スケジュール

**Reply by azu** [reply]
https://www.kadokawa.co.jp/product/302303004295/

**Reply by azu** [reply]
定価： 4,290円 （本体3,900円＋税）
発売日：2023年06月09日

**Reply by azu** [reply]
予約ができたら、

- メーリスで投げる
- 記事も書くか

発売日が来たら

- メール
- 記事を書く

**Comment by azu** [comment]
- レビューアーへの献本の宛先と流れ
- 献本依頼

**Reply by azu** [reply]
- フォーマット: PDF or epub or 物理本
- 物理の人は住所も一緒に
- GitHubアカウントも一緒に送ってもらう

**Reply by azu** [reply]
献本依頼 連絡つきそうな人には依頼してみる

**Comment by azu** [comment]
- #1658

**Comment by azu** [comment]
6/8(木) 19:00 ~
